service/history/replication/task_processor_manager.go

```diff
@@ -303,3 +316,21 @@ func (r *taskProcessorManagerImpl) cleanupReplicationTasks() error {
 	}
 	return err
 }
+
+func (r *taskProcessorManagerImpl) checkReplicationDLQSize() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	ctx = headers.SetCallerInfo(ctx, headers.SystemBackgroundCallerInfo)
```

Comment:
plz use `SystemPreemptableCallerInfo` for all replication related persistence call.

Reply:
Updated